ui/src/components/SkipNavigation/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { string } from 'prop-types';
+import { withStyles } from '@material-ui/core';
+import Button from '../Button';
+
+@withStyles({
+  skipButton: {
+    position: 'absolute',
+    left: '-999px',
```

Comment:
`left: -999,`

Reply:
Done. Mind if I ask what the difference between the two is? Thank you :D

Reply:
There is no difference except fewer characters and consistency througout the codebase. :)